### PR TITLE
Command line intro

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: command_line_intro/login.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# getting in and out
+
+To use our ruby-office1 system you must log in with a username and password. For this workshop you will have a virtual machine with your princeton netid and a public key.
+
+For our first exercise let's change the password by typing the following
```
Comment (escowles) [inline command_line_intro/login.md]: We should remove these two mentions of passwords.

File: command_line_intro/login.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## online help
```
Comment (escowles) [inline command_line_intro/login.md]: This section on `man` is great — but I was expecting info on SSH here.

File: command_line_intro/files.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+* Can collect many operations into single entities
+* Requires input
+* Produces output
+* Uses special characters
+* Has variables and state
+* Uses irritating syntax
+
+
+Additionally, as with programming languages, there are more shells than you’ll really care to learn. Among shells, bash is most widely used, so that is what we’ll use in this discussion. The [csh](), [tcsh](), [fish]()and [ksh]() shell types are also popular. Features of various shells are featured in the table below:
```
Comment (escowles) [inline command_line_intro/files.md]: Do we want to say which shells are commonly-used? I only see `sh` and `bash` installed on `ruby-office1`, for example.

File: command_line_intro/files.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Files and Directories
```
Comment (escowles) [inline command_line_intro/files.md]: This is a good overview of shells, but the title made me expect more stuff like ls/cd/mkdir/rm/cat/more/etc.

File: command_line_intro/README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+  1. [Getting In and Out of the System](login.md)
+  1. [File and Directory Commands](files.md)
+      * [File and directories](sec2/file_and_directory.md)
+  1. [Editing Files](editing_files.md)
+      * [emacs](sec3/emacs.md)
+      * [vi](sec3/vi.md)
+      * [nano](sec3/nano.md)
+      * [sed](sec3/sed.md)
+  1. [C Shell Interface](c_shell_interface.md)
```
Comment (escowles) [inline command_line_intro/README.md]: Shouldn't we look at sh/bash instead of csh?

File: command_line_intro/files.md (hunk 4 above)
Comment (kayiwa) [reply command_line_intro/files.md]: that's definitely a markdown error.... https://github.com/pulibrary/ruby_for_metadataists/blob/command_line_intro/command_line_intro/sec2/file_and_directory.md

**Comment by kayiwa** [comment]
added your requested changes. I feel the first 3 sections can be done one week but it is otherwise best thought of as a self driven reference for neophytes.

**Pull request merged**